File: tuikit/src/main/java/com/tencent/qcloud/tim/uikit2/component/gatherimage/DynamicChatUserIconView.java
package com.tencent.qcloud.tim.uikit2.component.gatherimage;

import com.tencent.qcloud.tim.uikit2.modules.message.MessageInfo;
import com.tencent.qcloud.tim.uikit2.utils.ScreenUtil;

public abstract class DynamicChatUserIconView extends DynamicLayoutView<MessageInfo> {

    private int iconRadius = -1;

    public int getIconRadius() {
        return iconRadius;
    }

    /**
     * 设置聊天头像圆角
     *
     * @param iconRadius
     */
    public void setIconRadius(int iconRadius) {
        this.iconRadius = ScreenUtil.getPxByDp(iconRadius);
    }
}

File: im/src/main/java/com/black/im/view/DynamicChatUserIconView.kt
package com.black.im.view

import com.black.im.model.chat.MessageInfo
import com.black.im.util.ScreenUtil


abstract class DynamicChatUserIconView : DynamicLayoutView<MessageInfo?>() {
    /**
     * 设置聊天头像圆角
     *
     * @param iconRadius
     */
    var iconRadius = -1
        set(iconRadius) {
            field = ScreenUtil.getPxByDp(iconRadius)
        }

}